usaspending_api/submissions/tests/test_load_dabs_submission_window_schedule.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+
+from datetime import datetime, timezone
+from model_mommy import mommy
+from django.core.management import call_command
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
+
+SCHEDULE_FILE = "usaspending_api/data/testing_data/test_dabs_submission_window_schedule.csv"
+
+FUTURE_DATE = datetime(9999, 12, 31, tzinfo=timezone.utc)
+CURRENT_REVEAL_DATE = datetime(2000, 1, 1, tzinfo=timezone.utc)
+
+
+@pytest.mark.django_db
+def test_schedule_is_updated(client):
+    mommy.make(
```

Comment:
For all of your Model Mommy statements can you provide more to the definition of `test_schedule_is_updated`? There aren't many fields on that table, but a field such as `is_quarter` could silently break tests in the future.

Reply:
This is no longer necessary now that logic based on `is_quarter` has been removed